diff --git a/config.def.h b/config.def.h
index 1edb647..369497d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -12,10 +12,14 @@ static const char *colors[SchemeLast][2] = {
 	[SchemeNorm] = { "#bbbbbb", "#222222" },
 	[SchemeSel] = { "#eeeeee", "#005577" },
 	[SchemeOut] = { "#000000", "#00ffff" },
+	[SchemeBorder] = { "#005577", "#000000" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
 
+/* size of border */
+static unsigned int bordersize = 2;
+
 /*
  * Characters not considered part of a word while deleting words
  * for example: " /?\"&[]"
diff --git a/dmenu.1 b/dmenu.1
index 323f93c..48dcf1c 100644
--- a/dmenu.1
+++ b/dmenu.1
@@ -12,6 +12,8 @@ dmenu \- dynamic menu
 .IR prompt ]
 .RB [ \-fn
 .IR font ]
+.RB [ \-bo
+.IR color ]
 .RB [ \-nb
 .IR color ]
 .RB [ \-nf
@@ -60,6 +62,9 @@ defines the prompt to be displayed to the left of the input field.
 .BI \-fn " font"
 defines the font or font set used.
 .TP
+.BI \-bo " color"
+defines the border color.
+.TP
 .BI \-nb " color"
 defines the normal background color.
 .IR #RGB ,
diff --git a/dmenu.c b/dmenu.c
index 40f93e0..7438559 100644
--- a/dmenu.c
+++ b/dmenu.c
@@ -25,7 +25,7 @@
 #define TEXTW(X)              (drw_fontset_getwidth(drw, (X)) + lrpad)
 
 /* enums */
-enum { SchemeNorm, SchemeSel, SchemeOut, SchemeLast }; /* color schemes */
+enum { SchemeNorm, SchemeSel, SchemeOut, SchemeBorder, SchemeLast }; /* color schemes */
 
 struct item {
 	char *text;
@@ -188,6 +188,10 @@ drawmenu(void)
 			drw_text(drw, mw - w, 0, w, bh, lrpad / 2, ">", 0);
 		}
 	}
+
+	drw_setscheme(drw, scheme[SchemeBorder]);
+	drw_rect(drw, 0, topbar ? bh-bordersize : 0, mw, bordersize, 1, 0);
+
 	drw_map(drw, win, 0, 0, mw, mh);
 }
 
@@ -715,7 +719,7 @@ setup(void)
 static void
 usage(void)
 {
-	die("usage: dmenu [-bfiv] [-l lines] [-p prompt] [-fn font] [-m monitor]\n"
+	die("usage: dmenu [-bfiv] [-l lines] [-p prompt] [-fn font] [-bo color] [-m monitor]\n"
 	    "             [-nb color] [-nf color] [-sb color] [-sf color] [-w windowid]");
 }
 
@@ -756,6 +760,8 @@ main(int argc, char *argv[])
 			colors[SchemeSel][ColBg] = argv[++i];
 		else if (!strcmp(argv[i], "-sf"))  /* selected foreground color */
 			colors[SchemeSel][ColFg] = argv[++i];
+		else if (!strcmp(argv[i], "-bo"))  /* border color */
+			colors[SchemeBorder][ColFg] = argv[++i];
 		else if (!strcmp(argv[i], "-w"))   /* embedding window id */
 			embed = argv[++i];
 		else
