diff --git a/components/battery.c b/components/battery.c
index 1c753f9..b3b7b48 100644
--- a/components/battery.c
+++ b/components/battery.c
@@ -56,10 +56,10 @@
 			char *state;
 			char *symbol;
 		} map[] = {
-			{ "Charging",    "+" },
-			{ "Discharging", "-" },
-			{ "Full",        "o" },
-			{ "Not charging", "o" },
+			{ "Charging",    "󰂏" },
+			{ "Discharging", "󰂌" },
+			{ "Full",        "󰁹" },
+			{ "Not charging", "󰂑" },
 		};
 		size_t i;
 		char path[PATH_MAX], state[12];
diff --git a/config.def.h b/config.def.h
index d805331..1773a5f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -4,7 +4,7 @@
 const unsigned int interval = 1000;
 
 /* text to show if no value can be retrieved */
-static const char unknown_str[] = "n/a";
+static const char unknown_str[] = "";
 
 /* maximum output string length */
 #define MAXLEN 2048
@@ -65,5 +65,11 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ temp,            "· %s°C",  "/sys/class/thermal/thermal_zone0/temp" },
+	{ disk_perc, " · DISK %s%%",  "/" },
+	{ ram_perc,   " · RAM %s%%",  NULL },
+	{ battery_state,    " · %s",  "BAT0" },
+	{ battery_perc,     " %s%%",  "BAT0" },
+	{ run_command,      " · %s",  "wpctl get-volume @DEFAULT_AUDIO_SINK@ | awk '{ if ($3 == \"[MUTED]\") { printf \"󰕿  󰟢  \" } else { printf \"󰕿%4s%\", 100*$2 } }'" },
+	{ datetime,         " · %s",  "%a %d/%m/%Y · %H:%M" },
 };
